File: Cliente.java
public class Cliente {
    private int id;

    private String nombre;

    private String direccion;

    public Cliente(int id, String nombre, String direccion) {
        this.id = id;
        this.nombre = nombre;
        this.direccion = direccion;
    }

    public int getId() {
        return id;
    }

    public String getNombre() {
        return nombre;
    }


    public String obtenerInformacion(){
        return String.format("Id: %d Nombre: %s y direccion: %s", id, nombre, direccion);
    }
    public boolean esNombreIgual(String nombre) {
        return this.nombre.equalsIgnoreCase(nombre);
    }

}

File: Cliente.py
class Cliente:
    def __init__(self, id_cliente, nombre, direccion):
        self.id_cliente = id_cliente
        self.nombre = nombre
        self.direccion = direccion

    def get_id(self):
        return self.id_cliente

    def get_nombre(self):
        return self.nombre

    def obtener_informacion(self):
        return f"Id: {self.id_cliente} Nombre: {self.nombre} Dirección: {self.direccion}"

    def es_nombre_igual(self, nombre):
        return self.nombre.lower() == nombre.lower()
